### ISSUE TITLE
Remove dependence on PyCall

### ISSUE THREAD

**Issue by charleskawczynski** [climate-machine/CalibrateEmulateSample.jl#18]
It looks like PyCall is only used to for the [wasserstein_distance](https://github.com/climate-machine/CalibrateEmulateSample.jl/blob/master/src/Histograms.jl#L199). It looks like we can call it from [EarthMoversDistance.jl](https://github.com/mirkobunse/EarthMoversDistance.jl)

**Comment by dburov190** [comment]
No, EarthMoversDistance.jl is a poor wrapper of an even poorer implementation from last century; I've tried it, it crashes every other time and is very slow. Python code works without hiccups.

**Comment by charleskawczynski** [comment]
I'd like to keep this issue open, since removing the dependence on PyCall would significantly reduce the build/development time-- even if that means translating python's [wasserstein_distance](https://github.com/scipy/scipy/blob/v1.4.1/scipy/stats/stats.py#L6934-L7008)

**Comment by odunbar** [comment]
I know I am late to the party here, but SKlearn also involves python calls. And GaussianProcesses.jl is again very slow (regards to optimization of hyperparameters) for when I tried this, so for now this will remain in the code too.

**Comment by charleskawczynski** [comment]
Agreed, I think this is more of a long term issue to resolve (if possible/feasible)

**Comment by jakebolewski** [comment]
I wonder if GaussianProcesses has matured somewhat in the meantime, is there a good comparative performance benchmark to try?
